<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Msg_Field_Selection</class>
 <widget class="QFrame" name="Msg_Field_Selection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="lineWidth">
   <number>0</number>
  </property>
  <widget class="QComboBox" name="cb_field">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_message">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Message</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_package">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_field">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Field</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_package">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Package</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_message">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
